static/js/compose_state.js

```diff
@@ -5,6 +5,10 @@ import * as compose_pm_pill from "./compose_pm_pill";
 let message_type = false; // 'stream', 'private', or false-y
 let recipient_edited_manually = false;
 
+// We use this variable to keep track of whether user has viewed the topic resolved
+// banner - so that we can show it to him only once
```

Comment:
Can you please use the gender neutral "them" pronoun instead of "him"?

Reply:
Thank you! Nice catch. I'm sorry, I'll try to be more inclusive going forward. Will do the change here and in the comment below as well.